baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -101,3 +102,15 @@ For example, if you have non-production requests that flow through the same serv
 ```
 baggage: isProduction=false
 ```
+# Mutating baggage
+A system receiving a `baggage` request header SHOULD send it to outgoing requests.
+A system MAY mutate the value of this header before passing it on.
+
+Because baggage entry keys, values, and metadata are not specified here, producers and consumers MAY agree on any set of mutation rules that don't violate the specification. For example, keys may be deduplicated by keeping the first entry, keeping the last entry, or concatenating values together.
```

Comment:
I'd suggest we recommend the "default" policy for deduplication. Maybe keeping last or keeping first are the best?